arches/app/views/api.py

```diff
@@ -389,7 +390,7 @@ def get(self, request, nodeid, zoom, x, y):
                             ) AS geom,
                             1 AS total
                         FROM geojson_geometries
-                        WHERE nodeid = %s and resourceinstanceid not in %s) AS tile;""",
+                        WHERE nodeid = %s and resourceinstanceid in %s) AS tile;""",
```

Comment:
Didn't realise I committed this line this is the hacky change I'm not confident in but I'm about to test if it has an impact